addon/components/grid-stack-item.js

```diff
@@ -65,6 +65,20 @@ export default class extends Component {
     }
   }
 
+  /**
+   * @method didUpdateAttrs
+   * @override
+   */
+  didUpdateAttrs() {
+    super.didUpdateAttrs(...arguments);
+
+    let parent = this.parentContainer;
+    if (parent && parent.gridStack) {
+      let { height, width, x, y } = this.options;
+      parent.gridStack.update(this.element, x, y, width, height);
```

Comment:
If we do this for `height, width, x, y` do we need to keep `attributeBindings` for `height, width, x, y`?

Reply:
I'm not sure there is any benefit in adding an exclusion list. It would be nice to remove the `attributeBindings` completely, since they are not supported in glimmer components.

Reply:
I think the benefit would be not updating the DOM multiple times (with `update()` & with `attributeBindings`) when `grid-stack-item` args update. It also would be nice to have a single path to set `x, y, width, height` (and all args).

> It would be nice to remove the attributeBindings

Totally agree :+1:

I spoke with a gridstack maintainer on slack, looks like direct attribute _updates_ are not officially supported. I filed an [issue](https://github.com/gridstack/gridstack.js/issues/1236) for this enhancement. Another issue is that not all `item` options are supported in the current `update()` method. I filed an [issue](https://github.com/gridstack/gridstack.js/issues/1235) for that too :). 

Given, the need for a larger refactor, I think we can merge this and clean up the flow separately.

Thanks for your work on this 👍

Reply:
I can fix the `GridStack.update()` to take all options (easy) but not sure I will get to (or should) track DOM changes to update internal state.